homesensorhub/sensors/types/pressure.py

```diff
@@ -0,0 +1,24 @@
+"""Module which contains the pressure type class."""
+from routing.payload import Payload
+from sensors.types.sensor_type import SensorType
+
+import datetime
+
+
+class Pressure(SensorType):
+    """Class which represents a type of value collected by a sensor."""
+
+    TYPE = 'Pressure'
+
+    def get_payload(self) -> Payload:
```

Comment:
move method to SensorType parent class

Reply:
Based on this commend I ended up changing the structure again. Ty. :+1: